scripts/actions/utils/docs-content-tools/i18n-exclusions.yml

```diff
@@ -13,4 +13,3 @@ excludePath:
     - src/content/docs/infrastructure/manage-your-data/data-instrumentation/host-integrations-metrics.mdx
 excludeType:
   ja-JP:
```

Comment:
if empty i believe you can just remove the whole shebang

Reply:
I'd rather keep it so I have the template in case we need to exclude types.

Reply:
since code runs on this, we should really remove this if not used.

Reply:
we can add some doc on this so it's clear what options are available and how to use it.